src/cdk/keycodes/modifiers.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+type ModifierKey = 'altKey' | 'shiftKey' | 'ctrlKey' | 'metaKey';
```

Comment:
should we use an enum for this?

Reply:
We could, but I wanted to keep the consumption as short as possible. With the enum people would need another import and the call would look like `hasModifierKeys(event, ModifierKey.altKey, ModifierKey.shiftKey)` which gets a little long.

Reply:
Strings also make sense here because they are the real properties on `KeyboardEvent`